screens/Trivia/index.js

```diff
@@ -1,5 +1,11 @@
-import React, { useState, useContext } from 'react';
-import { View, StyleSheet, ScrollView, TouchableOpacity } from 'react-native';
+import React, { useState, useContext, useEffect } from 'react';
+import {
+  ActivityIndicator,
```

Comment:
Added loading spinner for when it's retrieving category data and when you select a difficulty and the questions are being retrieved